staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 type Interface interface {
-	Add(item interface{})
+	Add(items ...interface{})
```

Comment:
this is a breaking change to the Interface, For breaking change, a few options: 1) send an email to kubernetes-sig-api-machinery@google.com 2) discuss it in biweekly sig-api-machinery meeting. 3) add to release notes about the breaking change.

Reply:
It is, I guess the question is how many external implementations do we expect. It won't break any callers, just parallel interface impls. I'll email the list though :)

Reply:
Probably be better to add `(q *Type) AddAll(items ...interface{})` and then provide default implementations.  Still changing the interface but won't impact clients.

Reply:
Also changing Add() can force a heap allocation if you only have one item (or it did at one point) so callers only sending one in high performance code may still prefer to use the single add

Reply:
Reasonable, I'll rework it :)